configs/components/puppet-runtime.json

```diff
@@ -1 +1 @@
-{"location":"http://builds.delivery.puppetlabs.net/puppet-runtime/201810110/artifacts/","version":"201810110"}
+{"location":"http://builds.delivery.puppetlabs.net/puppet-runtime/ff3a05c054651c1d1217deb8ebe3a82c9ae79f80/artifacts/","version":"201810110.1.gff3a05c"}
```

Comment:
You'll need to exclude changes to puppet-runtime.json from this PR, as this file is auto-incremented by our CI.